project2/styles.css

```diff
@@ -0,0 +1,238 @@
+* {
+    box-sizing: border-box;
+}
+
+body {
+    background-color: #508d8b;
+    color: black;
+    font-family: termina, sans-serif;
+}
+
+a {
+    text-decoration: none;
+}
+
+
+h2,
+h3,
+h4,
+p {
+    padding-left: 10px;
```

Comment:
You could increase the padding on the `.content` `div` and then avoid having to put padding on each of these individual elements and the `.img-wrapper` class, too.